hack/lib/golang.sh

```diff
@@ -828,6 +826,8 @@ kube::golang::get_physmem() {
 #   KUBE_BUILD_PLATFORMS - Incoming variable of targets to build for.  If unset
 #     then just the host architecture is built.
 kube::golang::build_binaries() {
+  kube::log::status "WARNING: linux/arm will no longer be built/shipped by default, please build it explicitly if needed."
```

Comment:
maybe move inside the `if [[ ${#platforms[@]} -gt 1 ]]; then` bit where we're building more than one platform? that'll keep this from showing up when building the single host platform?